src/backend/vector/ifma/edwards.rs

```diff
@@ -312,4 +312,53 @@ mod test {
         let P = &constants::ED25519_BASEPOINT_TABLE * &Scalar::from(8475983829u64);
         doubling_test_helper(P);
     }
+
+    #[test]
+    fn basepoint_odd_lookup_table_verify() {
+        use crate::backend::vector::ifma::constants::BASEPOINT_ODD_LOOKUP_TABLE;
+        use crate::constants;
+
+        let basepoint_odd_table =
+            NafLookupTable8::<CachedPoint>::from(&constants::ED25519_BASEPOINT_POINT);
+        println!("basepoint_odd_lookup_table = {:?}", basepoint_odd_table);
```

Comment:
I copied this from the AVX2 tests to have an equivalent check of the AVX512IFMA lookup table, but I don't have a suitable device to test this.